src/services/DeliveryEmployeeService.ts

```diff
@@ -0,0 +1,63 @@
+import axios, { AxiosResponse } from "axios";
+import { DeliveryEmployeeResponse } from "../models/DeliveryEmployeeResponse";
+import { DeliveryEmployeeRequest } from "../models/DeliveryEmployeeRequest";
+
+export const getDeliveryEmployees = async (): Promise<DeliveryEmployeeResponse[]> => {
+    try {
+        const response: AxiosResponse = await axios.get(`${process.env.API_URL}/delivery-employee`);
+
+        return response.data;
+    } catch (e) {
+        throw new Error('Failed to get delivery employees');
+    }
+}
+
+export const getDeliveryEmployee = async (id: String): Promise<DeliveryEmployeeResponse[]> => {
+    try {
+        const response: AxiosResponse = await axios.get(`${process.env.API_URL}/delivery-employee/${id}`);
+
+        return response.data;
+    } catch (e) {
+        throw new Error('Failed to get delivery employee');
```

Comment:
could we include their id in the error message so we know which one it failed to get, same for other error messages